capture/src/capture.cpp

```diff
@@ -113,7 +113,7 @@ int main( int argc, char** argv )
     const char* output = nullptr;
     int port = 8086;
     int seconds = -1;
-    int64_t memoryLimit = -1;
+    int64_t memoryLimit = tracy::NO_WORKER_MEMORY_LIMIT;
```

Comment:
What is the purpose of this change? How is writing or reading this much text more clear in intent than a negative value?